scripts/trafficlight.js

```diff
@@ -0,0 +1,128 @@
+/*
```

Comment:
please can we make scripts TS too :(

Reply:
How do i do that? I was just copying the existing practice in the scripts folder for everything to be a .js script

Reply:
we can either run everything through `tsc` or pick up `ts-node` as a thing.

I'm keen to stop the proliferation of JS in the project, which means we shouldn't be producing more JS scripts.

Reply:
I believe i've set this up by cribbing off existing bits and pieces of the code;